### DISCUSSION TITLE
INNR BY 266

### DISCUSSION THREAD

**Discussion by julianfs** [Koenkk/zigbee2mqtt#12985] [Devices]
Hello all,

Whilst previously having no problems with Z2M & INNR BY 266 bulbs, I now have 3 out of 6 with issues. I'm using HA core-2022.6.7, Mosquitto broker (6.1.2), Zigbee2MQTT (1.25.2-1) and a Conbee II (fw v2.16.1).

Paring works fine, but after pairing any brightness or ON/Off commands end in timeout errors. Deleting the bulb from Z2M puts it back in paring mode, so there is some comms.

If I pair these problematic bulbs with an Alexa, I can control them without issues.

Could this be a Z2M issue or a conbee issue? I 'feels' like some old reference to the devices is remaining after deletion & causing problems are re-adding. I'm not having issues with any Hive zigbee bulbs.

Thanks for any advice.

**Comment by shanetuohy** [comment]
Having this same issue. Did you ever solve it @julianfs ? 

They seemed to work originally, but very quickly stopped working. Deleting and repairing works fine and the device appears to be ok, but cannot control the devices.

**Reply by shanetuohy** [reply]
After pairing, the last debug message I get from the device seems to be that the device is asking for an OTA. I wonder if that has something to do with it? 

Also, when generating a network map, LQI succeeds, does this imply that the coordinator is able to communicate with the device? 

```
debug 2023-01-11 11:33:57Received Zigbee message from '0xc49886000007f789', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":1,"fileVersion":587273489,"imageType":557,"manufacturerCode":4454}' from endpoint 1 with groupID null
debug 2023-01-11 11:33:57Device '0xc49886000007f789' requested OTA
debug 2023-01-11 11:51:20LQI succeeded for '0xc49886000007f789'
```

**Reply by shanetuohy** [reply]
Debug logs from a failed brightness update. 
```

debug 2023-01-11 12:10:14Received MQTT message on 'zigbee2mqtt/0xc49886000007f789/set' with data '{"brightness":82}'
debug 2023-01-11 12:10:14Publishing 'set' 'brightness' to '0xc49886000007f789'
error 2023-01-11 12:11:15Publish 'set' 'brightness' to '0xc49886000007f789' failed: 'Error: Command 0xc49886000007f789/0 genLevelCtrl.moveToLevelWithOnOff({"level":82,"transtime":0}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'
debug 2023-01-11 12:11:15Error: Command 0xc49886000007f789/0 genLevelCtrl.moveToLevelWithOnOff({"level":82,"transtime":0}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received) at DeconzAdapter.sendZclFrameToEndpoint (/app/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:658:23) at runMicrotasks (<anonymous>) at runNextTicks (node:internal/process/task_queues:61:5) at processTimers (node:internal/timers:499:9) at Endpoint.sendRequest (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:299:20) at Endpoint.command (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:772:28) at Object.convertSet (/app/node_modules/zigbee-herdsman-converters/converters/toZigbee.js:968:13) at Publish.onMQTTMessage (/app/lib/extension/publish.ts:246:36)
info 2023-01-11 12:11:15MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'brightness' to '0xc49886000007f789' failed: 'Error: Command 0xc49886000007f789/0 genLevelCtrl.moveToLevelWithOnOff({\"level\":82,\"transtime\":0}, {\"sendWhen\":\"immediate\",\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (no response received)'","meta":{"friendly_name":"0xc49886000007f789"},"type":"zigbee_publish_error"}'
```

**Reply by shanetuohy** [reply]
I have downgraded my Conbee II to the firmware recommended on the wiki (0x26580700) as I was on a newer firmware that may have been causing issues. 

Latest stable Z2M

Home Assistant running on Home Assistant Blue hardware.

**Reply by shanetuohy** [reply]
When power cycling the light, it does announce itself on the network, but same behaviour persists. Any command sent to it times out after 60 seconds. 
```

debug 2023-01-11 12:20:11 Device '0xc49886000007f789' announced itself
info 2023-01-11 12:20:11 MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xc49886000007f789","ieee_address":"0xc49886000007f789"},"type":"device_announce"}'
info 2023-01-11 12:20:11MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0xc49886000007f789"},"type":"device_announced"}'
debug 2023-01-11 12:20:13 Retrieving state of '0xc49886000007f789' after reconnect
error 2023-01-11 12:21:14 Failed to read state of '0xc49886000007f789' after reconnect (Read 0xc49886000007f789/0 genOnOff(["onOff"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received))
```

**Comment by julianfs** [comment]
Hi, 

Interesting to hear you've had the same issue.

I didn't find a solution, only a workaround. I have a 2nd USB conbee II running ZHA. I moved the INNR bulbs from MQTT to ZHA and they've been rock solid since & haven't tried moving back.

Both sticks are on the same firmware, so this indicates to me that it is a software issue, rather than coordinator?

Are you saying above that moving to the latest stable release fixes your issue or it still remains?

**Reply by shanetuohy** [reply]
No, downgrading didn't fix the problem. 

Give you have this working on ZHA, seems to me this is an issue in Z2M? 

I'll upgrade this to an issue and see if we can get this solved or debugged more.

**Comment by shanetuohy** [comment]
Issue created on this @julianfs  https://github.com/Koenkk/zigbee2mqtt/issues/16100

**Comment by julianfs** [comment]
See #18961 for the fix.
